docs/source/manual/auth.rst

```diff
@@ -216,3 +216,41 @@ resource method.
 If you have a resource which is optionally protected (e.g., you want to display a logged-in user's
 name but not require login), you need to implement a custom filter which injects a security context
 containing the principal if it exists, without performing authentication.
+
+Testing Protected Resources
+===========================
+
+Add this dependency into your ``build.gradle`` file:
+
+.. code-block:: java
```

Comment:
I'm not sure what `Maven syntax` you are referring to is. Please point me to some docs. Thanks.